tests/phpunit/AutoReview/Makefile/MakefileTest.php

```diff
@@ -231,26 +197,14 @@ public function test_the_test_target_runs_all_the_tests(): void
         $this->assertEqualsCanonicalizing($rootTestTargets, $testPrerequisites);
     }
 
-    public function test_the_docker_test_target_runs_all_the_tests(): void
```

Comment:
This test was to ensure the (phony) `test-docker` executes all (phony) tests, i.e. that`test-docker` executes:

- `autoreview`
- `test-unit-docker`
- `test-e2e-docker`
- `test-infection-docker`
- and if added any `test-x-docker`

Reply:
here we have a problem I think, because we have `autoreview` (cs, phpstan, etc.) and it's pair `autoreview-docker`. 

At the same time we have `test-autoreview` (which runs PHPUnit with autoreview group) and its par `test-autoreview-docker`.

So this convention is not followed. We should either rename `autoreview` command to something else or update/remove this test?